webapp-django/crashstats/signature/jinja2/signature/signature_report.html

```diff
@@ -13,21 +13,28 @@
 {% block content %}
 <div id="mainbody">
     <div class="page-heading">
+        <a href="#" class="display-toggle-filters toggle-filters show">Show Search Filters</a>
         <h2>Signature report for <em>{{ signature }}</em></h2>
+        <p>
+            Showing results from
+            <time class="ago" datetime="{{ query.start_date.isoformat() }}">{{ query.start_date | human_readable_iso_date }} UTC</time>
+            to <time class="ago" datetime="{{ query.end_date.isoformat() }}">{{ query.end_date | human_readable_iso_date }} UTC</time>.
+            <a href="#" class="toggle-filters">Edit filters</a>.
```

Comment:
I don't like this. Sorry :)

It's confusing that it's labelled so differently from the "Show Search Filters". (ie. "edit" != "filter"). 
I think it's enough to have the "Show Search Filters" link in the upper right-hand corner.